src/components/Profile/Skills.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import EditSkills from '../CustomMaterialComponents/EditSkills';
+
+type SkillsCompProps = {
+    isEditMode : boolean;
+    setFreelancer: Function;
+    freelancer: any;
+    skills: string[];
+    setSkills: Function;
+}
+
+const Skills = ({isEditMode, setFreelancer, freelancer, skills, setSkills}:SkillsCompProps) => {
+  useEffect(() => {
+    setFreelancer({ ...freelancer, skills: skills.map((skill) => { return { name: skill } }) })
+  }, [skills])
+
+    return (
+        <div className="mx-[30px] lg:mx-[40px]">
+        <div className="header-editable">
+          <h5>Skills</h5>
+        </div>
+        {
+          isEditMode
+            ? (
+              <EditSkills {...{ skills, setSkills }} />
+            )
+            : (
+              <div className="flex flex-wrap gap-[20px] mt-[24px]">
+                {freelancer?.skills?.map?.((skill: any, skillIndex: string) => (
```

Comment:
`skillIndex: string` 🤔

Reply:
I think the index used in the map callback are string. No?